js/modules/data.js

```diff
@@ -663,39 +663,30 @@ async function removeAllSessions(id) {
 function setifyProperty(data, propertyName) {
   if (!data) return data;
   const returnData = data;
```

Comment:
Currently this function mutates the input parameter `data`. You'll need to take a copy:
```suggestion
  const returnData = Object.assign({}, data);
```
It probably doesn't affect the code right now but can lead to surprises in the future ;)

Reply:
```suggestion
  const returnData = { ...data };
```
Shortcut to object.assign